assets/xml/objects/object_ossan.xml

```diff
@@ -3,7 +3,7 @@
         <Animation Name="gObjectOssanAnim_000338" Offset="0x338"/>
         <Texture Name="gOssanEyesTLUT" OutName="ossan_eyes_tlut" Format="rgba16" Width="168" Height="1" Offset="0x4728"/>
         <Texture Name="gOssanEyeOpenTex" OutName="eye_open" Format="ci8" Width="32" Height="32" Offset="0x4878"/>
-        <Texture Name="gOssanBeardTex" OutName="beard" Format="ci8" Width="32" Height="32" Offset="0x4C78"/>
+        <Texture Name="gOssanBeardTex" OutName="beard" Format="ci8" Width="32" Height="32" Offset="0x4C78" TlutOffset="0x4728"/>
         <Texture Name="gOssanEyeHalfTex" OutName="eye_half" Format="ci8" Width="32" Height="32" Offset="0x52B8"/>
         <Texture Name="gOssanEyeClosedTex" OutName="eye_closed" Format="ci8" Width="32" Height="32" Offset="0x56B8"/>
```

Comment:
missing tluts, probably 0x4728. Maybe should be renamed if gOssanEyes is TLUT for beard too

Reply:
This object seems mostly unfinished so I'll probably just leave it for now.